src/components/RestaurantDetails/components/Info/Info.js

```diff
@@ -0,0 +1,95 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import infoReader from './readers/infoReader';
+
+//Images
+import star from './images/star.svg';
+import delivery from './images/delivery.svg';
+import rupee from './images/rupee.svg';
+
+//Components
+import OtherInfo from './components/OtherInfo';
+
+//CSS
+import './Info.css';
+
+function Info(props) {
```

Comment:
rename Info to something more meaningful, like which Info this component is showing